app/UI/app_update.tsx

```diff
@@ -598,11 +599,33 @@ export async function handle_SendMessage(
         }
         events = events_send;
     } else if (args.navigationModel.activeNav == "Public") {
-        const nostr_event = await prepareNormalNostrEvent(ctx, {
-            content: event.text,
-            kind: NostrKind.TEXT_NOTE,
-            tags: generateTags(event.text, args.getEventByID, args.current_relay.url),
-        });
+        let replyToEvent: NostrEvent | undefined;
+        if (event.reply_to_event_id) {
```

Comment:
here we have UI event and nostr event. Rename `event` to `ui_event` to be more readable.